runtime/testing/e2e/a11y.ts

```diff
@@ -0,0 +1,46 @@
+/*jslint node: true */
+'use strict';
+
+const AxeBuilder = require('axe-webdriverjs');
+import { browser } from 'protractor';
+
+export class A11yTest {
+
+  public static log(results): any {
+      //console.error(`${violations}`);
+      var numResults = results.violations.length;
+
+      if (numResults > 0) {
+        console.error("Accessibility failure(s) found for: " + results.url + "\n");
+        results.violations.forEach(result => {
+          var label = result.nodes.length === 1 ? 'element' : 'elements';
+          var msg = result.nodes.reduce(function(msg, node) {
+              return `${msg}  Location: ${node.target[0]}
+  ${node.html}`;
+          }, "\n");
+          console.error(`${result.nodes.length} ${label} failed '${result.id}' rule: ${result.help} ${msg}
+  Get help at: ${result.helpUrl}
+`)
+        });
+      }
+  }
+
+  public static run(): Promise<Number> {
```

Comment:
Could be something to look at if teams need something like this as they start using. Maybe even to turn off tests for some reason. axe itself only run sthe tests it finds elements for that meet the criteria (i.e., only runs test for labels if there are labels in the doc/selector).